### ISSUE TITLE
Document availability of request.auth.token in verifyFunc()

### ISSUE THREAD

**Issue by jmm** [dwyl/hapi-auth-jwt2#184]
[`request.auth.token` is set](https://github.com/dwyl/hapi-auth-jwt2/blob/d570a28966832e22f4704c7f034162e5d2a216ff/lib/index.js#L38) by the time [`verifyFunc()`](https://github.com/dwyl/hapi-auth-jwt2/blob/d570a28966832e22f4704c7f034162e5d2a216ff/lib/index.js#L81) is invoked. Can that be documented? Current documentation only refers to it as being available after validation.

**Comment by nelsonic** [comment]
Well spotted @jmm (_again_!) Documentation update?

**Comment by jmm** [comment]
Thanks! This...

> The *encoded* JWT (token) is extracted from the headers of the request

...and all similar references should just say something to the effect of "extracted from the request", since additional token sources (cookie, URL) were added. (Technically reading it from a cookie is still extracting it from the headers, but for the purposes of these docs I think it's more helpful to treat headers and cookies as fundamentally different places to read the token from.) Same with references to the `Authorization` header, since the header to read from is configurable. I opened #190 for that.
